cloudinit/analyze/show.py

```diff
@@ -217,10 +219,11 @@ def gather_timestamps_using_dmesg():
                 # systemd wont start cloud-init in this case,
                 # so we cannot get that timestamp
                 return SUCCESS_CODE, kernel_start, kernel_end, kernel_end
-
-    except Exception:
-        pass
-    return TIMESTAMP_UNKNOWN
+    except (ValueError, subp.ProcessExecutionError):
```

Comment:
> Do you know why we want to ignore these two?

`ProcessExecutionError` when `dmesg` fails

`ValueError` when `float()` cast fails (also catches `UnicodeError` if dmesg contains invalid utf-8 )

> Nit: Since both except arms return the same thing, should the return be moved outside?

+1, works for me